compiler/parser/parser.peg

```diff
@@ -445,7 +445,8 @@ SampleProc
           "keys": ARRAY(MAP("kind": "Assignment",
                    "lhs": MAP("kind": "ID", "name": "shape"),
                    "rhs": MAP("kind": "Call", "name": "typeof",
-                              "args": ARRAY(e)))),
+                              "args": ARRAY(e),
+                              "where": NULL))),
```

Comment:
Maybe add `"where": NULL` to the `"kind": "Call"` maps up in the `TBD` rule to avoid surprises should someone resurrect that rule?

Reply:
Why does that `TBD` rule still exist?